docs/ml/deploy/tflite_cpu.md

```diff
@@ -125,6 +125,13 @@ The following parameters are available for a `"tflite_cpu"` model:
 | `label_path` | Optional | The absolute path to a `.txt` file that holds class labels for your TFLite model, as a `string`. This text file should contain an ordered listing of class labels. Without this file, classes will read as "1", "2", and so on. |
 | `num_threads` | Optional | An integer that defines how many CPU threads to use to run inference. Default: `1`. |
 
+{{% alert title="Note" color="note" %}}
+If you **Deploy Model on Robot**, `model_path` and `label_path` will be automatically configured in the format `"${packages.<model_name>}/<model-name>.tflite"`.
+
+If you take the **Path to existing model on robot** approach, your model and label paths do not have to be in the same format.
+They can look like `home/models/fruit/my_fruit_model.tflite`.
```

Comment:
Nit: Using `example` to avoid seemingly like a recommendation.

```suggestion
For example, they might resemble `home/models/fruit/my_fruit_model.tflite`.
```